api/parsers/fiat.go

```diff
@@ -0,0 +1,37 @@
+package parsers
+
+import (
+	"strings"
+
+	"github.com/gin-gonic/gin"
+)
+
+type currencyFilter struct {
```

Comment:
Maybe would be a nice idea left those structs within the parsers public to allow a third-party library to reuse them.